generator/mavgen_python.py

```diff
@@ -795,18 +765,7 @@ def parse_char(self, c):
 
         self.total_bytes_received += len(c)
 
-        if self.native is not None:
-            if native_testing:
-                self.test_buf.extend(c)
-                m = self.__parse_char_native(self.test_buf)
-                m2 = self.__parse_char_legacy()
-                if m2 != m:
-                    logger.error("Native: %s\\nLegacy: %s\\n", m, m2)
-                    raise Exception("Native vs. Legacy mismatch")
-            else:
-                m = self.__parse_char_native(self.buf)
-        else:
-            m = self.__parse_char_legacy()
+        m = self.__parse_char_legacy()
```

Comment:
Thanks for this PR!

Not sure if it makes sense to still call it `__parse_char_legacy`, instead of renaming it to something like `__parse_char` or `__parse_char_internal`.